src/test/java/org/reactome/release/downloaddirectory/GenerateGOAnnotationFile/BiologicalProcessAnnotationBuilderTest.java

```diff
@@ -65,59 +54,101 @@ public void biologicalProcessAnnotationWithCatalystLineBuilderTest() throws Exce
-        Mockito.when(mockCatalystInst.getAttributeValue(ReactomeJavaConstants.physicalEntity)).thenReturn(mockCatalystPEInst);
-        Mockito.when(GOAGeneratorUtilities.isValidCatalystPE(mockCatalystPEInst)).thenReturn(true);
-        Mockito.when(mockCatalystPEInst.getSchemClass()).thenReturn(mockPESchemaClass);
-        Mockito.when(GOAGeneratorUtilities.isMultiInstancePhysicalEntity(mockPESchemaClass)).thenReturn(true);
-        Mockito.when(mockPESchemaClass.isa(ReactomeJavaConstants.EntitySet)).thenReturn(true);
-        Mockito.when(mockCatalystPEInst.getAttributeValuesList(ReactomeJavaConstants.hasMember)).thenReturn(mockMemberSet);
-        Mockito.when(mockCatalystPEMemberInst.getSchemClass()).thenReturn(mockMemberSchemaClass);
-        Mockito.when(mockMemberSchemaClass.isa(ReactomeJavaConstants.EntityWithAccessionedSequence)).thenReturn(true);
-        Mockito.when(mockCatalystPEMemberInst.getAttributeValue(ReactomeJavaConstants.referenceEntity)).thenReturn(mockReferenceEntityInst);
-        Mockito.when(mockCatalystPEMemberInst.getAttributeValue(ReactomeJavaConstants.species)).thenReturn(mockSpeciesInst);
-        Mockito.when((GOAGeneratorUtilities.isValidProtein(mockReferenceEntityInst, mockSpeciesInst))).thenReturn(true);
-        Mockito.when(((GKInstance) mockSpeciesInst.getAttributeValue(ReactomeJavaConstants.crossReference))).thenReturn(mockCrossReferenceInst);
-        Mockito.when(mockCrossReferenceInst.getAttributeValue(ReactomeJavaConstants.identifier)).thenReturn("1234");
-        Mockito.when(mockReactionInst.getReferers(ReactomeJavaConstants.hasEvent)).thenReturn(mockEventReferralSet);
-        Mockito.when(mockEventReferralInst.getAttributeValuesList(ReactomeJavaConstants.goBiologicalProcess)).thenReturn(mockGOBioProcessSet);
-        Mockito.when(mockGOBioProcessInst.getAttributeValue(ReactomeJavaConstants.accession)).thenReturn("1234");
-        Mockito.when(mockEventReferralInst.getAttributeValue(ReactomeJavaConstants.stableIdentifier)).thenReturn(mockStableIdentifierInst);
-        Mockito.when(mockStableIdentifierInst.getAttributeValue(ReactomeJavaConstants.identifier)).thenReturn("A1234");
-        Mockito.when(mockReferenceEntityInst.getAttributeValue(ReactomeJavaConstants.identifier)).thenReturn("R1234");
-        Mockito.when(GOAGeneratorUtilities.getStableIdentifierIdentifier(mockEventReferralInst)).thenReturn("1234");
-        Mockito.when(GOAGeneratorUtilities.getSecondaryIdentifier(mockReferenceEntityInst)).thenReturn("R5678");
-        Mockito.when(GOAGeneratorUtilities.generateGOALine(mockReferenceEntityInst, GOAGeneratorConstants.BIOLOGICAL_PROCESS_LETTER, "GO:1234", "REACTOME:1234", GOAGeneratorConstants.TRACEABLE_AUTHOR_STATEMENT_CODE, "1234")).thenCallRealMethod();
+        Mockito.when(mockReactionInst.getAttributeValuesList(ReactomeJavaConstants.catalystActivity))
+            .thenReturn(mockCatalystSet);
```

Comment:
Breaking an expression into >=2 lines just because it seems long makes it really hard to track differences. This is functionally identical to https://github.com/reactome/release-download-directory/pull/16/files#diff-b0b20393e7b87945812bbb32a552b0fa3f7dac47f983d2861799a24b5869bc67L68 but the diff shows it as changed, and I can't even ignore this with "hide whitespace changes"

Reply:
Okay, I'll try to limit these line changes to their own PRs when possible so it won't obscure logic changes.

Reply:
Would you consider only doing it to lines that you are already editing for some other reason? It creates a lot of noise in the PR. Maybe doing a separate PR just for formatting, if you really feel it's necessary to reformat large parts of code.

Reply:
Sure, for a PR that I'm adding or changing logic around I'll try to restrict formatting changes.  The changes do help me to read the code, so maybe I'll do a first pass on code that isn't mine for a "format" PR.